snuba/web/query.py

```diff
@@ -215,7 +223,7 @@ def _dry_run_query_runner(
     return QueryResult(
         {"data": [], "meta": []},
         {
-            "stats": {},
+            "stats": {"storage_names": storage_names},
```

Comment:
What's the reason for this change?

Reply:
I was thinking about having a preloaded set of storages a user can choose from if they want to get tracing data. Assuming we're given a non-composite query, we could return a list of probable storages they would choose from? I don't know how useful it would be though so I can remove this now and re-add it if we want to pursue this route in another tech debt week.